test/isImplementedCache.test.ts

```diff
@@ -33,9 +33,14 @@ describe('ImplementedCache', () => {
     it('should return an event that has skillId', () => {
       isImplementedCache.add(event, true, 'smapleSkill');
 
-      expect(isImplementedCache.get(event)).toBeDefined();
-      expect(isImplementedCache.get(event)?.skillId).toBeDefined();
-      expect(isImplementedCache.get(event)?.skillId).toBeString();
+      expect(isImplementedCache.get(event,'smapleSkill')).toBeDefined();
+      expect(isImplementedCache.get(event,'smapleSkill')?.skillId).toBeDefined();
+      expect(isImplementedCache.get(event,'smapleSkill')?.skillId).toBeString();
```

Comment:
You could actually go so far to make sure that skill is equal to `smapleSkill`

Reply:
updated